Dockerfile-develop

```diff
@@ -62,6 +61,21 @@ RUN cat octopus-configlog-warnings
 # all in one line to make image smaller
 RUN make && make install && make clean && make distclean
 
+RUN echo "DEBUG starts here XXXX ----------------"
+RUN echo "`octopus` cannot find libsym:"
+RUN ldd /usr/local/bin/octopus
+RUN ldd /usr/local/bin/octopus | grep libsym
+
+RUN echo "The problem is solved by setting LD_LIBRARY_PATH - but that's probably not intended?"
+ENV LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
This is the line that works around the problem described in https://github.com/fangohr/octopus-in-docker/issues/9 . 

While it would be better to fix #9 in Octopus, this work around allows us to keep providing a Docker image for now.